server/src/main/java/org/elasticsearch/index/IndexMode.java

```diff
@@ -225,6 +225,75 @@ public void validateSourceFieldMapper(SourceFieldMapper sourceFieldMapper) {
             }
         }
 
+        @Override
+        public boolean isSyntheticSourceEnabled() {
+            return true;
+        }
+    },
+    LOGS("logs") {
+        @Override
+        void validateWithOtherSettings(Map<Setting<?>, Object> settings) {}
```

Comment:
I think we should still validate:

```
 settingRequiresTimeSeries(settings, IndexMetadata.INDEX_ROUTING_PATH);
            settingRequiresTimeSeries(settings, IndexSettings.TIME_SERIES_START_TIME);
            settingRequiresTimeSeries(settings, IndexSettings.TIME_SERIES_END_TIME);
```

?

Reply:
Yes I will validate this and add a test checking we have an error if we try to use those settings when using "logs"